djaoapp/templates/saas/users/roles.html

```diff
@@ -27,64 +27,56 @@
       </form>
     </user-typeahead>
     </div>
-    <div class="col-sm-9">
+    <div class="col-sm-6">
         <div class="form-row">
-            {% include "saas/_filter.html" %}
+            {% include "saas/_filter_input.html" %}
         </div>
     </div>
     </div>
-    <table class="table table-striped">
-      <thead>
-        <tr>
-          <th>Name
-            <a href="#" @click="sortBy('slug')"><i :class="sortIcon('slug')"></i></a>
-          </th>
-          <th>Role
-            <a href="#" @click="sortBy('role_name')"><i :class="sortIcon('role_name')"></i></a>
-          </th>
-          <th>Created at
-            <a href="#" @click="sortBy('created_at')"><i :class="sortIcon('created_at')"></i></a>
-          </th>
-          <th></th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr v-show="!itemsLoaded">
-          <td colspan="3">
-            <h3  class="text-center"><i class="fa fa-refresh fa-spin fa-2x"></i></h3>
-          </td>
-        </tr>
-        <tr v-show="itemsLoaded && items.results.length == 0" v-cloak>
-          <td colspan="3">No organization accessible for {{user}}</td>
-        </tr>
-        <tr :id="entry.slug" v-for="(entry, index) in items.results"
-           v-show="itemsLoaded && items.results.length > 0" v-cloak>
-          <td>
-            <a :href="'{{urls.profile_base}}' + entry.slug + '/'">[[entry.printable_name]]</a>
-          </td>
-          <td class="text-center">
-            [[entry.request_key ? "requested" : entry.role_description.title]]
-          </td>
-          <td class="text-center">
-            [[entry.created_at | formatDate]]
-          </td>
-          <td style="vertical-align:middle;">
-            <button class="btn btn-danger btn-sm"
+  <div class="row">
+      <div class="col-12 col-md-6">
+        <div v-show="!itemsLoaded">
+            <h3 class="text-center"><i class="fa fa-refresh fa-spin fa-2x"></i></h3>
+        </div>
+        <div v-show="itemsLoaded && items.results.length == 0" v-cloak>
+            <p class="text-muted mt-2">No items</p>
+        </div>
+      </div>
+  </div>
+  <div class="row my-4">
+    <div class="col-lg-6 col-12 my-1" v-for="(item, index) in items.results" v-show="itemsLoaded && items.results.length > 0" v-cloak>
+        <div class="card organization-card">
+          <img :src="item.picture ? item.picture : '{{'/static/img/default-user.png'|djasset}}'" class="card-img-top" :alt="item.printable_name">
```

Comment:
Maybe we need a `default-organization.png` here to make it easier to visually understand the page.